### PR TITLE
Add cryptsetup package

### PR DESCRIPTION
**Pull request by bernard-wagner**
## What does this do

Makes cryptsetup available in the default console and during boot. Allowing for LUKS encrypted devices. 

## How was this tested

Followed build instructions to build RancherOS for AMD64. Booted ISO using docker-machine and add second harddrive. Encrypted second harddrive using the following procedure:
```
[root@rancher-test ~]# cryptsetup -y -v luksFormat --force-password /dev/sdb

WARNING!
========
This will overwrite data on /dev/sdb irrevocably.

Are you sure? (Type uppercase yes): YES
Enter passphrase for /dev/sdb:
Verify passphrase:
Command successful.
[root@rancher-test ~]# cryptsetup luksOpen /dev/sdb luks-sdb
Enter passphrase for /dev/sdb:
[root@rancher-test docker]# ls /dev/mapper/
control    luks-sdb

[root@rancher-test docker]# uname -a
Linux rancher-test 4.14.73-rancher #1 SMP Thu Oct 4 00:19:21 UTC 2018 x86_64 GNU/Linux
```

### CODE REVIEW

**Comment by niusmallnan** [comment]
related to https://github.com/rancher/os/issues/2551

**Comment by bernard-wagner** [comment]
@niusmallnan Can you provide an update on what is needed for the PR to get merged?

File: config/amd64/buildroot-config [hunk 1]
```diff
@@ -856,7 +856,7 @@ BR2_PACKAGE_ACPID=y
 # BR2_PACKAGE_CDRKIT is not set
 # BR2_PACKAGE_CRYPTSETUP is not set
```
Comment (niusmallnan) [inline config/amd64/buildroot-config]: I think you should change this line.

**Comment by niusmallnan** [comment]
@bernard-wagner  Can you update? I want to add this feature to v1.5.0.

File: config/amd64/buildroot-config (hunk 1 above)
Comment (bernard-wagner) [reply config/amd64/buildroot-config]: Will push commit tonight

**Comment by niusmallnan** [comment]
Also, need to synchronize with the buildroot menuconfig, It doesn't matter, I will finish it.

**Pull request merged**